docs/developer-docs/integrations/sns/testing/testing-locally.md

```diff
@@ -28,91 +29,119 @@ These scripts have been most tested with a stack that looks like this:
 * A single canister that is being decentralized.
 * A canister that can be deployed to local replica via `dfx deploy`.
 
-## Testing via `sns-testing` repo
-To use the `sns-testing` commands and their arguments properly, please consult the `sns-testing` [README](https://github.com/dfinity/sns-testing#sns-lifecycle).
+## Getting started
+
+To use the `sns-testing` commands and their arguments properly, you need to first set up the `sns-testing` repo properly. Please see [README](https://github.com/dfinity/sns-testing#sns-lifecycle).
 
 :::info
-If you have started using `sns-testing` before August 2023 and are using the old legacy flow of launching an SNS (please see [here](../launching/index.md) for more context), please continue your work with the legacy documentation on [this old README](https://github.com/dfinity/sns-testing/blob/v1-legacy/README.md). In particular, 
+* All of the commands in this article require `sns-testing` to be set up properly. 
```

Comment:
Is it really the case that the commands here will basically be run "on top of `sns-testing`"? 
I thought here we would just describe the commands in terms of dfx commands (not relying on sns-testing as a tool) 
I assume that some of the setup that was done in sns-testing is also needed for this to work though but I am not quite sure which part.

Reply:
yes,

1. you need to follow the install README of `sns-testing`. 

2. `dfx` commands are not enough, there is quite a bit of boiler plate within the bash scripts.